### DISCUSSION TITLE
Patch information - request for more cheat?

### DISCUSSION THREAD

**Discussion by Sxri** [3kh0/ext-remover#97] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Hello, my school uses some stupid extension called "Boarderless classroom" for teachers to view our screens and manually block websites, close our tabs, etc. I used to be able to disable the extension with ingot and the exploit.js until the patch of course. anyways, it was popular among our school for a long time.

This is boarderless classrooms website if you'd like to take a peek at it, this is the only thing they use for unblocking. and their wifi too, but i use my hotspot for that: https://www.aristotlek12.com/

 Any exploits to mess with this or anything? Thanks!

**Comment by ghost** [comment, marked as answer]
how does it work? does it redirect you when you open a blocked tab? is it an overlay?

**Reply by lurklite** [reply]
redirects to their chrome extention page!

**Reply by lurklite** [reply]
this is me on my alt. it basically shows a blue screen and redirects. the url has the chrome extention id in it too.

**Reply by The-Man-Is-Me** [reply]
this works for gogardian, worth a try
javascript:(function(){let url = prompt("about:blank spoofer | what website do you want to spoof");win = window.open();win.document.body.style.margin = '0';win.document.body.style.height = '100vh';var iframe = win.document.createElement('iframe');iframe.style.border = 'none';iframe.style.width = '100%';iframe.style.height = '100%';iframe.style.margin = '0';iframe.src = url;win.document.body.appendChild(iframe);})();
Also https://website-aio.herokuapp.com/

**Reply by Sxri** [reply]
> this works for gogardian, worth a try javascript:(function(){let url = prompt("about:blank spoofer | what website do you want to spoof");win = window.open();win.document.body.style.margin = '0';win.document.body.style.height = '100vh';var iframe = win.document.createElement('iframe');iframe.style.border = 'none';iframe.style.width = '100%';iframe.style.height = '100%';iframe.style.margin = '0';iframe.src = url;win.document.body.appendChild(iframe);})(); Also https://website-aio.herokuapp.com/

the script doesnt work it still gets blocked and that site is blocked and flagged for "proxy"

**Reply by md4r2** [reply]
@Sxri u could deploy ur own instance of the site

**Reply by Sxri** [reply]
> @Sxri u could deploy ur own instance of the site

no clue how to do that. the main goal was disabling the extension / making them not be able to see our screens or modify our computers. but unblocking any site without a proxy would be sick too. 

Idk if thats possible but yeah

**Reply by md4r2** [reply]
Check the [wiki](https://github.com/holy-unblocker/website-aio/wiki) for the site for how to deploy. Also ye that would be so sickk

**Reply by Sxri** [reply]
> Check the [wiki](https://github.com/holy-unblocker/website-aio/wiki) for the site for how to deploy. Also ye that would be so sickk

I don´t need a proxy, i have my own i made, my own website, my own everything. I just need a method to mess with the school board, thats it

**Reply by synapt0** [reply]
> this works for gogardian, worth a try javascript:(function(){let url = prompt("about:blank spoofer | what website do you want to spoof");win = window.open();win.document.body.style.margin = '0';win.document.body.style.height = '100vh';var iframe = win.document.createElement('iframe');iframe.style.border = 'none';iframe.style.width = '100%';iframe.style.height = '100%';iframe.style.margin = '0';iframe.src = url;win.document.body.appendChild(iframe);})(); Also https://website-aio.herokuapp.com/

You think this could work for securly?

**Reply by The-Man-Is-Me** [reply]
@NotJantix posibly, give it a try

**Reply by synapt0** [reply]
@The-Man-Is-Me Didn't work sadly

**Reply by Alex2Mayhem** [reply]
> > this works for gogardian, worth a try javascript:(function(){let url = prompt("about:blank spoofer | what website do you want to spoof");win = window.open();win.document.body.style.margin = '0';win.document.body.style.height = '100vh';var iframe = win.document.createElement('iframe');iframe.style.border = 'none';iframe.style.width = '100%';iframe.style.height = '100%';iframe.style.margin = '0';iframe.src = url;win.document.body.appendChild(iframe);})(); Also https://website-aio.herokuapp.com/
> 
> You think this could work for securly?

How does this code work?/?

**Reply by Alex2Mayhem** [reply]
> 

How does this work it just opens an about blank page???

**Reply by Alex2Mayhem** [reply]
> this works for gogardian, worth a try javascript:(function(){let url = prompt("about:blank spoofer | what website do you want to spoof");win = window.open();win.document.body.style.margin = '0';win.document.body.style.height = '100vh';var iframe = win.document.createElement('iframe');iframe.style.border = 'none';iframe.style.width = '100%';iframe.style.height = '100%';iframe.style.margin = '0';iframe.src = url;win.document.body.appendChild(iframe);})(); Also https://website-aio.herokuapp.com/

Doesn't work, it just puts you in an about:blank page. plus you didn't give instructions
